lib/watir_drops/page_object.rb

```diff
@@ -97,8 +119,27 @@ def fill_form(model)
     def inspect
       '#<%s url=%s title=%s>' % [self.class, url.inspect, title.inspect]
     end
+
     alias selector_string inspect
 
+
+    def on_page?
+      begin
```

Comment:
Fun trick, you don't need `begin`/`end` if you are rescuing the entire method. just make the rescue statement flush with the `def`.